src/test/resources/org/broadinstitute/hellbender/engine/hg19mini.fasta

```diff
@@ -0,0 +1,804 @@
+>1 dna:chromosome chromosome:GRCh37:1:1:16000:1
```

Comment:
This is a dupe of a file already checked in under `src/test/resources` -- please delete.